packages/remix-forms/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "remix-forms",
+  "version": "0.17.0",
+  "description": "Magically create forms + actions in Remix!",
+  "main": "./dist/index.js",
+  "module": "./dist/index.mjs",
+  "types": "./dist/index.d.ts",
```

Comment:
@danielweinmann , I changed it from `./lib` to `./dist` to match the turborepo's default and I think it is a nice and valid pattern as well